src/NUnitFramework/tests/Internal/TestNameGeneratorTests.cs

```diff
@@ -86,6 +89,14 @@ public string ParameterizedTests(string pattern, object[] args)
             return new TestNameGenerator(pattern).GetDisplayName(_simpleTest, args);
         }
 
+        [TestCase("{m}{p}", new object[] { 1 }, ExpectedResult = "TestMethodWithArgs(a: 1, b: ?)")]
```

Comment:
If the `b` parameter was optional, I think I'd expect `TestMethodWithArgs(a: 1)`.